acmeissuer.go

```diff
@@ -198,6 +202,9 @@ func NewACMEIssuer(cfg *Config, template ACMEIssuer) *ACMEIssuer {
 	if template.Logger == nil {
 		template.Logger = DefaultACME.Logger
 	}
+	if template.HTTPProxy == nil {
+		template.HTTPProxy = http.ProxyFromEnvironment
```

Comment:
We should get this default value from `DefaultACME`, otherwise if the user sets `DefaultACME.HTTPProxy`, it will have no effect.

```suggestion
		template.HTTPProxy = DefaultACME.HTTPProxy
```